focus-ios/focus-ios-tests/XCUITest/CopyPasteTest.swift

```diff
@@ -0,0 +1,130 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class CopyPasteTest: BaseTestCase {
+    // Smoketest
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/395745
```

Comment:
The automation status is not updated in TR. It shows Automation disabled

Reply:
Fixed. 🙂